src/components/WeatherWidget/WeatherWidget.jsx

```diff
@@ -0,0 +1,132 @@
+import { StyledWrapper, StyledWrapperWeather } from "./WeatherWidget.styles";
+import { useState, useEffect } from "react";
+import GetLocation from "../../services/location";
+import GetWeather from "../../api/weather";
+import CircularProgress from "@mui/material/CircularProgress";
+import Box from "@mui/material/Box";
+import { Switch, FormControlLabel } from "@mui/material";
+
+const rainyImage = process.env.PUBLIC_URL + "/images/rainy-day.png";
+const sunnyImage = process.env.PUBLIC_URL + "/images/sun.png";
+const snowyImage = process.env.PUBLIC_URL + "/images/snow.png";
+const windImage = process.env.PUBLIC_URL + "/images/wind.png";
+const cloudImage = process.env.PUBLIC_URL + "/images/cloud.png";
```

Comment:
What if this became an object too:

```suggestion
const iconsMappings = {
  rain: "rainy-day.png",
  sun: "sun.png"
}
```

and so then on line 68 you can do...